lib/backend/k8s/resources/globalbgppeer_test.go

```diff
@@ -123,9 +118,4 @@ var _ = Describe("Global BGP conversion methods", func() {
 		Expect(kvp.Value).To(BeAssignableToTypeOf(&model.BGPPeer{}))
 		Expect(kvp.Value).To(Equal(kvp1.Value))
 	})
-
-	It("should fail to convert an invalid Kuberenetes resource", func() {
```

Comment:
because it was essentially testing `ResourceNameToIP(t.Metadata.Name)`  in the conversion code https://github.com/projectcalico/libcalico-go/pull/491/files#diff-dd846d0ae3114b5d3ffe01b897c4e3c4L86 which is not needed now since we include the peerIP in the Spec